static/app/views/issueList/types.tsx

```diff
@@ -21,3 +21,11 @@ export type GroupSearchView = {
   querySort: IssueSortOptions;
   id?: string;
 };
+
+// Id is always returned by the response
+export type GroupSearchViewResponse = {
+  id: string;
+  name: string;
+  query: string;
+  querySort: IssueSortOptions;
+};
```

Comment:
we tend to not put "Response" in our type names, even though i definitely do on my personal projects.

Reply:
oh i see you've got two types and trying to move them over?

Reply:
maybe do `Required<GroupSearchView>`

Reply:
That is exactly what i was looking for but couldn't find, thanks!